WalletWasabi.Tests/UnitTests/SmartBlockProviderTests.cs

```diff
@@ -42,7 +44,12 @@ public async void TestAsync()
 
 		private class MockProvider : IBlockProvider
 		{
-			public Func<uint256, CancellationToken, Task<Block>> OnGetBlockAsync { get; set; }
+			public Func<uint256, CancellationToken, Task<Block>> OnGetBlockAsync { get; }
```

Comment:
Property should be placed under the constructor.

Reply:
Fixed.